src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurDetailLineGroupForCollector.java

```diff
@@ -100,12 +100,24 @@ protected void addCorpCardPersonalExpenseDetailLine(ConcurStandardAccountingExtr
     protected Map<String, List<ConcurStandardAccountingExtractDetailLine>> getConsolidatedPersonalExpensesMapForLine(
             ConcurStandardAccountingExtractDetailLine detailLine) {
         if (detailLine.getJournalAmount().isPositive()) {
-            if (collectorHelper.lineRepresentsReturnOfCorporateCardPersonalExpenseToUser(detailLine)) {
-                return consolidatedCorpCardPersonalExpenseCredits;
-            } else {
-                return consolidatedCorpCardPersonalExpenseDebits;
-            }
-        } else if (collectorHelper.lineRepresentsReturnOfCorporateCardPersonalExpenseToUniversity(detailLine)) {
+            return getConsolidatedPersonalExpensesMapForDebitLine(detailLine);
+        } else {
+            return getConsolidatedPersonalExpensesMapForCreditLine(detailLine);
+        }
+    }
+
+    protected Map<String, List<ConcurStandardAccountingExtractDetailLine>> getConsolidatedPersonalExpensesMapForDebitLine(

```

Comment:
The line passed to this method is expected to be a debit one, which was why I named the method as such (and similarly for the credit method). These methods are configured to return a mismatched map if the line is a return/credit of a personal expense, since that allows for properly deducting the returned amount from the current personal charges when the similar transactions get wrapped up.